.github/workflows/filtering-ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"] # , "macos-latest", "windows-latest"
-        python-version: ["3.11"]
```

Comment:
In an ideal world you'd use a matrix like:

https://github.com/theislab/pertpy/blob/main/.github/workflows/test.yml#L20